src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAlone.java

```diff
@@ -80,4 +80,18 @@ void method6(int a) {
 
     public @interface TestAnnottation4 { String value();
     }
+
+    interface Interface1
+    {
+        int i = 1;
+        public void meth1(); } //violation
+
+    interface Interface2
+    { int i = 1; public void meth1(); } //violation
+
+    interface Interface3 {
+        void display();
+        interface Interface4 {
+            void myMethod();
+        }} //violation - for both of the right curly braces
```

Comment:
please make comments here and above with space - `// violation`

Reply:
Above violation comments didn't have spaces, so I followed that. I'll add spaces in all of my comments then.